src/selectedtests/app/app.py

```diff
@@ -42,7 +42,7 @@ def create_app(mongo: MongoWrapper, evg_api: EvergreenApi) -> Flask:
     return app
 
 
-def main():
+def main() -> Flask:
```

Comment:
I believe this ticket was originally prioritized below MAKE-1085 (switch to fastAPI), since a chunk of typing will probably change with that ticket.

Reply:
My bad, I took the most straightforward looking ticket I saw in order to take a break from cedar stuff. The Web API layer is pretty slim so hopefully wont be too bad.